assets/tests/components/propertiesPanel/minischedule.test.tsx

```diff
@@ -178,6 +178,33 @@ const midRouteSwingPiece2: Piece = {
   endMidRoute: false,
 }
 
+const midRouteSwingWithNonRevFirst: Piece = {
+  runId: "run2",
+  blockId: "block",
+  startTime: 120,
+  startPlace: "swingplace",
+  trips: [
+    {
+      ...midRouteSwingTrip2,
+      routeId: null,
+      startPlace: "DH start location",
+      endPlace: "DH end location",
+    },
+    {
+      ...midRouteSwingTrip2,
+      startTime: midRouteSwingTrip2.endTime + 60,
+      endTime: midRouteSwingTrip2.endTime + 120,
+    },
+  ],
+  endTime: 480,
+  endPlace: "terminal1",
+  startMidRoute: {
+    time: 180,
+    trip: midRouteSwingTrip1,
+  },
+  endMidRoute: false,
+}
```

Comment:
**suggestion:** consider making a factory for this - we use Fishery and there are some existing factories located in `assets/tests/factories`.

Reply:
Experimenting/learning fishery here https://github.com/mbta/skate/tree/kb-kf-minischedule-fix-/fishery-factories